### DISCUSSION TITLE
Does fpPS4 can open or extract or install PKG files?

### DISCUSSION THREAD

**Discussion by ghost** [red-prig/fpPS4#60] [Q&A] [answered]
I got Undertale CUSA09415 today because yesterday I saw that Undertale is playable with fpPS4 and I want to give fpPS4 a try but Undertale appears to be a single PKG file maybe because Undertale is a small game and small games are usually distributed as a single PKG file.

I already know that small and tiny games can be distributed as PKG files.

In the past I also got Assault Heroes for the PS3, another small game, also as a single PKG file and in RPCS3 I could start playing it perfectly after I installed that PKG file.

In RPCS3 installing PKG files is very easy by just clicking "Files" and then clicking "Install Packages/Raps/Edats" and finally just browse to the pkg file that you want to install and play.

I got fpPS4 0.0.1 yesterday and it appears to be a cmd program right now and double clicking it in Windows File Explorer shows it for a half second before it disappears in my screen.

I tried to find page with information about how to use this cmd program with the commands that can be used by this program right now but I couldn't find anything.

I am not really sure that fpPS4 can right now install PKG files like RPCS3 but I don't really know.

**Comment by red-prig** [comment, marked as answer]
Go to the discord indicated on the main page, there are tutorials.

**Reply by ghost** [reply]
> Go to the discord indicated on the main page, there are tutorials.

Thank you very much! I didn't know that fpPS4 has a discord! I like discord! I will ask my question there then.

**Comment by ghost** [comment]
You can use PkgEditor and click "Export to GP4 project"

**Reply by ghost** [reply]
> You can use PkgEditor and click "Export to GP4 project"

I have followed the How_to_Setup_fpPs4_emulator.pdf from the fpps4-setup-tutorial discord channel just to learn how to extract the pkg file and then I have successfully played Undertale only with the -e command line which I saw in a comment or reply to some issue there.

Finally I have successfully played a PS4 title only with my PC for the first time!

fpPS4 resembles Mednafen a lot.

As how DuckStation is good to play PS1, PCSX2 to play PS2 and RPCS3 to play PS3 on your PC I believe that fpPS4 is going be the best emulator to play PS4 on your PC finally after 10 years.

Noting that fpPS4 started as good as how RPCS3 did and how much good is RPCS3 today so imagine how wonderful fpPS4 is going to be in the next years?

Surely that I will have to use Cheat Engine to cheat on PS4 games currently until fpPS4 will support native cheats in the future though I saw that the ps4-game-mods discord channel already exists so maybe fpPS4 can already do this?

**Reply by ghost** [reply]
You can use fpPS4 Temmie's Launcher as well which makes the startup process easier and not a hassle.

**Comment by ghost** [comment]
@buruto I actually found out not only small and tiny games can be made as PKGs, I heard in https://github.com/red-prig/fpPS4/issues/52 that even God of War Ragnarök can be packaged as a PKG file

**Comment by fighter1860** [comment]
Do you plan on making a ps5 emulator after making fpps4 later on?

**Reply by red-prig** [reply]
I don't see the point in talking about it now
